test/images/mount-tester/mt.go

```diff
@@ -232,39 +234,32 @@ func readWriteNewFile(path string, perm os.FileMode) error {
 	return readFileContent(path)
 }
 
-func readFileContentInLoop(path string, retryDuration int) error {
+func readFileContentInLoop(path string, retryDuration int, checkInitialContent bool) error {
 	if path == "" {
 		return nil
 	}
-	var content []byte
-	content = testFileContent(path, retryDuration)
-
-	fmt.Printf("content of file %q: %v\n", path, string(content))
-
+	testFileContent(path, retryDuration, checkInitialContent)
 	return nil
 }
 
-func testFileContent(filePath string, retryDuration int) []byte {
+func testFileContent(filePath string, retryDuration int, checkInitialContent bool) {
 	var (
 		contentBytes []byte
 		err          error
 	)
-
 	retryTime := time.Second * time.Duration(retryDuration)
 	for start := time.Now(); time.Since(start) < retryTime; time.Sleep(2 * time.Second) {
 		contentBytes, err = ioutil.ReadFile(filePath)
-		if err == nil {
+		if err != nil {
+			fmt.Printf("Error read file %s: %v, retry", filePath, err)
+			return
+		}
+		if checkInitialContent {
 			//Expected content "mount-tester new file\n", length 22
-			if len(contentBytes) == 22 {
-				break
-			} else {
+			if len(contentBytes) != 22 {
 				fmt.Printf("Unexpected length of file: found %d, expected %d.Retry", len(contentBytes), 22)
 			}
-		} else {
-			fmt.Printf("Error read file %s: %v, retry", filePath, err)
 		}
-
+		fmt.Printf("content of file %q: %v\n", filePath, string(contentBytes))
```

Comment:
I agree that this is an explicit change in functionality but as far as I know none is using  `file_content_in_loop` flag (or at least I couldn't find any).

My goal here is to keep to display the content of the file until the timeout. Not to match the initial content. I'm just trying to offer the opportunity to check the initial content as well (since this opportunity was alreadsy there).